src/main/java/uk/gov/pay/directdebit/mandate/model/Mandate.java

```diff
@@ -69,24 +69,24 @@ public void setState(MandateState state) {
         this.state = state;
     }
 
-    public MandateBankStatementReference getMandateReference() {
-        return mandateReference;
+    public MandateBankStatementReference getMandateBankStatementReference() {
```

Comment:
Should this maybe return an `Optional`? (Possibly not a change for now)

Reply:
Yes it should but it was existing and didn't want to play around too much at the same time. I'll either add it to this PR after addressing your comment above or in a future one.